packages/button/src/MultiButton.tsx

```diff
@@ -139,11 +139,13 @@ const StyledIcon = styled(ChevronDown)<StyledIconProps>`
 type ButtonProps = {
   label: string;
   value: string;
+  alwaysEnable?: boolean;
 };
 
 type Props = {
   mainButton: ButtonProps;
   secondaryButtons: Array<ButtonProps>;
+  enableSecondary?: boolean;
```

Comment:
Trengs det to properties for å støtte dette? Det burde være nok å definere alwaysEnable på sekundærknappen for at den skal være aktiv.

Reply:
Dette gjør at knappene i dropdown kan styres forskjellig slik at "lagre" er disabled og "lagre som ny versjon" ikke er det. Men brukeren får jo status på om artikkel er lagret fra hovedknappen, så er kanskje ikke noe problem at "lagre"-knappen er tilgjengelig i dropdown.

Reply:
Eg berre tenkte at den eine kunne utledes av den andre. At dersom sekundærknappen er markert som alltid-lagre så vil nedtrekksknappen være aktiv.

Reply:
Det var lurt, jeg skriver det om